flow/core/experiment.py

```diff
@@ -98,6 +101,13 @@ def run(self, num_runs, rl_actions=None, convert_to_csv=False):
         convert_to_csv : bool
             Specifies whether to convert the emission file created by sumo
             into a csv file
+        partition_name: str
+            Specifies the S3 partition you want to store the output file,
+            will be used to later for query. If NONE, won't upload output
+            to S3.
+        only_query: str
+            Specifies whether queries should be automatically run the
+            simulation data when it gets uploaded to s3
```

Comment:
I've updated the docstring, sorry for the confusion. I think this might be useful in the future if we only want to run the lastest model the new simulation data.